app-of-apps/templates/apps.yaml

```diff
@@ -19,6 +19,11 @@ spec:
     path: {{ .path }}
     repoURL: {{ $.Values.repoURL }}
     targetRevision: {{ coalesce .targetRevision $.Values.subChartsRevision "HEAD" | quote }}
+    {{- if .helmValues }}
+    helm:
+      values: |-
+{{ .helmValues | toYaml | indent 8 }}
```

Comment:
Can you please re-indent this line using also `{{-` and `nindent`? Thanks a lot!